src/components/Wizard/Step1.test.tsx

```diff
@@ -1,11 +1,7 @@
-//test routing, zie SOKtabs in aip
-//
-
 import * as React from 'react';
 import { screen } from '@testing-library/react';
 import Step1 from './Step1';
 import renderWithProviders from '~/tests/utils/withProviders';
-/* eslint-disable react/display-name */
 
 function renderComponent() {
 	return renderWithProviders(
```

Comment:
```
const renderComponent = () =>
	renderWithProviders(
		<Step1 getPostUrl={jest.fn()} getHeaders={jest.fn()} onFileSuccess={jest.fn()} onFileRemove={jest.fn()} />,
		{},
	);
```

Reply:
What's wrong with a named function? I've recently reverted to employing named functions as much as possible to aid in readable stack traces when exceptions are thrown.

Reply:
Nothing wrong with. but you're writing more lines of code than necessary IMHO.